README.md

```diff
@@ -4,18 +4,35 @@ CLI for creating sets of responsive images for the web
 
 ## Usage
 
-    resize {file or directory} [-r] [space delimited set of widths]
+```
+Usage:  resize [options]
```

Comment:
if these `-{option}` args are to be added, please ensure backwards compatibility so that the original style of command still works